MIQ-Booking-Assistance/miq-assistance.js

```diff
@@ -110,14 +132,33 @@ async function prepareAndCheckPage(page) {
     }, month);
 
     if (found) {
-        console.log("AVAILABLE! Found at: " + new Date().toLocaleString())
+        const status = "AVAILABLE! Found at: " + new Date().toLocaleString();
+        console.log(status)
+        updateElectronStatus('status-count', status);
     } else {
         // reload if nothing was available
-        process.stdout.moveCursor(0, -1) // up one line
-        process.stdout.clearLine(1) // from cursor to end
```

Comment:
Do you think there might be a way to keep using this instead if the script is run without Electron? Not a big deal, if it stays like that though.

Reply:
I'll try